<!-- -->
<launch>
    <node pkg="disparity_expansion" exec="disparity_conv" name="disparity_conv"/>
    <node pkg="disparity_expansion" exec="disparity_expansion" name="disparity_expansion">
	<param name="lut_max_disparity" value="180" />
        <param name="robot_radius" value="1.0" />
        <param name="bg_multiplier" value="2.0" />
        <param name="sensor_pixel_error" value="0.5" />
    </node>
</launch>
